app/api/controllers/AuditHandler.scala

```diff
@@ -40,7 +42,8 @@ object AuditHandler {
     pathParams = pathParams,
     queryParams = queryParams,
     requestBody = requestBody,
-    responseBodyMap = if (includeResponse) identity else _ => None
+    responseBodyMap = if (includeResponse) identity else _ => None,
+    version = version
   )
 
```

Comment:
Not related to your PR, but it would be good if the AuditHandler constructor could be made private (i.e. `case class AuditHandler private (.....)`) so that the above apply method must always be used